PatientMonitoringSystem/PatientMonitoringSystem/Controllers/BedsideSystemController.cs

```diff
@@ -17,11 +18,17 @@ public class BedsideSystemController
 
 		private readonly int maxModulesPerBedsideSystem;
 
+		private Timer poller;
+
 		public BedsideSystemController(BedsideSystemView bedsideSystemView, Guid bedsideSystemId)
 		{
 			this.bedsideSystemView = bedsideSystemView;
 			bedsideSystem = Program.BedsideSystems.Single(bs => bs.BedsideSystemId == bedsideSystemId);
 			maxModulesPerBedsideSystem = int.Parse(ConfigurationManager.AppSettings["MaxModulesPerBedsideSystem"]);
+			poller = new Timer();
```

Comment:
@harrytpstudent The timer will need to be disposed when the form is closed.

Reply:
If we want the values to keep being generated in the background, then we need this to stay.  We can turn the timer off when there are no modules present so we don't try and poll modules that aren't there.